README.md

```diff
@@ -60,7 +60,19 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct. This tables describes shows overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
+
+|                   |LVM                    |Direct              |
+|:--                |:--                    |:--                 |
+|Purpose            |avoids fragmentation   |simpler, somewhat faster, but storage may get fragmented   |
+|What is served     |LVM Logical Volume     |pmem block device   |
+|Region-affinity    |yes:One Volume Group per Region  |yes: namespace can belong to one Region only  |
```

Comment:
"Region-affinity" is another technical term which needs a footnote.

Why the special spelling of "One Volume Group per Region"? I would stick to plain English as much as possible. "yes: one volume group per region" looks fine to me.